kong/pdk/plugin.lua

```diff
@@ -6,6 +6,15 @@
 local _plugin = {}
 
 
+---
+-- Returns the instance ID of the running plugin.
```

Comment:
I assume this function only makes sense when being used inside a plugin, as we do sometimes use PDK functions in core code as well, let's add into the docs here to avoid confusion.

Reply:
Not sure if it's accurate to say "the plugin" instead of "the running plugin".